### PR TITLE
Add tutorial to join metadata

### PR DESCRIPTION
**Pull request by kelshmo**
This PR adds a tutorial article to the possible-future portal documentation site to describe how to merge individual, biospecimen and assay metadata files. This tutorial will also be published in the AD Knowledge Portal whitepaper - therefore, seeking feedback on: 
- readability? 
- code - are too many assumptions made for this data set in particular (e.g. metadata file naming conventions are parsable)? 
- is content missing?

For greater readability, [article is staged here](https://kelshmo.github.io/portalsDocs/articles/managing-metadata.html).

### CODE REVIEW

**Comment by kelshmo** [comment]
I merged this to stage the article for greater viz - @karawoo and @Aryllen I requested you to be a collaborator on this repo. It would be great to get your thoughts on the above points!

**Comment by Aryllen** [comment]
I will take a look at this in closer detail next week.

File: _articles/managing-metadata.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: Managing Metadata
+layout: article
+excerpt: Merge individual, biospecimen and assay-specific metadata files.
+category: tutorials
+---
+
+File annotations and metadata provide critical information about the data in the AD Knowledge portal. Annotations describe content within the files, including: type of data and assay, tissue of origin, source study, ID of the source individual and type of data. Annotations conform to a structured schema to make data highly searchable and standardized across studies and projects. The standards for defining, managing and implementing controlled vocabularies for content in Synapse is available in the synapseAnnotations Github repository. 
+
+Some types of annotations contain private health information, and cannot be represented as file annotations. For this reason, some of the metadata is stored within a set of csv files. There are typically several metadata csv files required to fully represent details of a single file or individual.
```
Comment (Aryllen) [inline _articles/managing-metadata.md:10]: > Some types of annotations contain private health information, and cannot be represented as file annotations. For this reason, some of the metadata is stored within a set of csv files.

In AD, this is not the only reason why all metadata is not annotated with. By stating this as being the reason for metadata files existing, it makes it seem like all the data in those metadata files is PHI, which is untrue. I would estimate that most metadata in these files is not PHI, but rather just extra details. People may not want to download the metadata if they think they will be accountable for PHI or if they simply don't believe they need any PHI details for their analysis. Additionally, that first sentence refers to metadata as annotations but then says that they aren't annotations. It's a little confusing.

> There are typically several metadata csv files required to fully represent details of a single file or individual.

While true, this also would need to be used in conjunction with annotations if trying to get the metadata about the file itself. For example, joining the metadata will tell a user who a specimen came from and what assay was done on it, but it won't tell the user which file the resulting assay data is in.

Suggestion:
While a subset of metadata is stored as annotations on the file, the complete metadata is stored within a set of csv files. The complete metadata expands on the annotation set with extra details about the individuals, specimens from the individuals, and the assay performed on the specimens. There are typically several metadata csv files required to fully represent the details of a single individual, specimen, or assay. These can also be combined with the file annotations to get the complete details about the file.

File: _articles/managing-metadata.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+---
+title: Managing Metadata
+layout: article
+excerpt: Merge individual, biospecimen and assay-specific metadata files.
+category: tutorials
+---
+
+File annotations and metadata provide critical information about the data in the AD Knowledge portal. Annotations describe content within the files, including: type of data and assay, tissue of origin, source study, ID of the source individual and type of data. Annotations conform to a structured schema to make data highly searchable and standardized across studies and projects. The standards for defining, managing and implementing controlled vocabularies for content in Synapse is available in the synapseAnnotations Github repository.
```
Comment (Aryllen) [inline _articles/managing-metadata.md:8]: > Annotations conform to a structured schema to make data highly searchable and standardized across studies and projects.

Both metadata and annotations conform to the structured schema. Not sure where to put that in this paragraph, but both are mentioned in the first sentence.

> synapseAnnotations Github repository

Could use a link here.

File: _articles/managing-metadata.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+Some types of annotations contain private health information, and cannot be represented as file annotations. For this reason, some of the metadata is stored within a set of csv files. There are typically several metadata csv files required to fully represent details of a single file or individual. 
+
+The individual metadata file describes each individual in the study. Each row corresponds to a unique individual identifiable by the key *individualID*. The biospecimen metadata file describes the specimens that were collected and includes metadata like tissue type, cell type or nucleic acid source. While each row corresponds to a unique specimen identified by the key *specimenID*, there may be multiple specimens per individual. The biospecimen file also contains the key *individualID*, thus the individual metadata file can be joined on this key. The assay metadata file contains experimental processing details such as technology used, batch processing, quality control metrics and the key *specimenID*. There may be more than one assay metadata file if specimens or individuals were characterized across several assay types.
```
Comment (Aryllen) [inline _articles/managing-metadata.md:12]: A bullet list or table might be easier for processing this paragraph. It is necessary, valuable information, but feels dense.

File: _articles/managing-metadata.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+# Merge Metadata Files on *individualID* and *specimenID*
+ 
+The following example merges the metadata files from the [*MC-CAA* study in the Alzheimer's Disease (AD) Knowledge Portal](https://adknowledgeportal.synapse.org/Explore/Data?QueryWrapper0=%7B%22sql%22%3A%22SELECT%20*%20FROM%20syn11346063%22%2C%22limit%22%3A25%2C%22offset%22%3A0%2C%22selectedFacets%22%3A%5B%7B%22concreteType%22%3A%22org.sagebionetworks.repo.model.table.FacetColumnValuesRequest%22%2C%22columnName%22%3A%22study%22%2C%22facetValues%22%3A%5B%22MC-CAA%22%5D%7D%2C%7B%22concreteType%22%3A%22org.sagebionetworks.repo.model.table.FacetColumnValuesRequest%22%2C%22columnName%22%3A%22dataSubtype%22%2C%22facetValues%22%3A%5B%22metadata%22%5D%7D%5D%7D). See the [Downloading Data Programmatically article]({{ site.baseurl }}{% link _articles/download-data.md %}#R) for steps to download data from the portal in R. In this example, we join the same individual and biospecimen file to both the RNAseq and SNP array assay metadata files. You can think of the assay files as containing a subset of the individuals or biospecimens.
```
Comment (Aryllen) [inline _articles/managing-metadata.md:18]: I would also add a link to the main study page in the portal, as well. The metadata files can be found on that page, along with information about the study itself.

>  You can think of the assay files as containing a subset of the individuals or biospecimens.

This sentence is confusing. I understand what you are trying to say, but a user may not. I would leave this out or say something more to the affect of "this study has two assay types, which means the RNAseq assay was done on a subset of biospecimens and SNP array assay was done on the rest." That's a cumbersome sentence, though.

**Review by Aryllen — COMMENTED** [review]
I know this was already merged, but commenting only on the non-code sections since I am not sure how you would prefer feedback.

File: _articles/managing-metadata.md (hunk 2 above)
Comment (kelshmo) [reply _articles/managing-metadata.md:8]: metadata doesn't always conform to the structured schema. I will add the link. thanks!

**Pull request merged**